src/main/java/carpet/CarpetSettings.java

```diff
@@ -509,6 +509,13 @@ public String description() {
     )
     public static int fillLimit = 32768;
 
+    @Rule(
+            desc = "Customizable structure block limit",
+            options = {"48", "124", "256"},
```

Comment:
Why did you have the middle option be 124 instead of 128?
```suggestion
            options = {"48", "128", "256"},
```

Reply:
Oh, that's a typo, sorry for that.